esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -378,40 +391,49 @@ fi
 # To reset: remove appropriate section from Apache config
 if [[ "$MODE_APACHE" || "$MODE_ALL" ]]
 then
+    # TODO(benkraft): we should put each site in a separate conf file, and just
+    # make sure they all get imported.
+    # TODO(benkraft): we should attempt to factor much of the shared config out
+    # into a single file used by all sites, so it's easier to update.
     cat >>$APACHE_CONF_FILE <<EOF
+
 #   $INSTITUTION $GROUPNAME (automatically generated)
-WSGIDaemonProcess $SITENAME processes=4 threads=1 maximum-requests=1000
+WSGIDaemonProcess $SITENAME processes=2 threads=1 maximum-requests=500 display-name=${SITENAME}wsgi
 <VirtualHost *:80 *:81>
     ServerName $ESPHOSTNAME
-    ServerAlias $SITENAME-orig.learningu.org
 
-    #   Redirect to be used for failover
-    # RedirectMatch (.*) http://$SITENAME-backup.learningu.org\$1
+    Include $APACHE_REDIRECT_CONF_FILE
 
     #   Caching - should use Squid if performance is really important
-    # CacheEnable disk /
-
-    #   Redirect HTTP requests to HTTPS for security
-    RewriteEngine On
-    RewriteCond %{HTTP:X-Forwarded-Proto} !^https
-    RewriteCond %{REQUEST_URI} !^/media
-    RewriteRule ^(.*)$ https://%{SERVER_NAME}%{REQUEST_URI} [R,L]
+    CacheEnable disk /
 
     #   Static files
     Alias /media $BASEDIR/esp/public/media
     Alias /static $BASEDIR/esp/public/static
 
+    <Location /media>
```

Comment:
You took out the ``dav on`` line, but doesn't this entire section configure dav? Should everything else here also be removed?